### ISSUE TITLE
undefined: ginSwagger.Config && undefined: ginSwagger.CustomWrapHandler

### ISSUE THREAD

**Issue by zhangyile** [swaggo/gin-swagger#65]
Issues:
```bash
$  go run main.go
# command-line-arguments
./main.go:28:13: undefined: ginSwagger.Config
./main.go:32:25: undefined: ginSwagger.CustomWrapHandler

```

below is some debug info
```go
// cat go.mod
module godemo

go 1.12

require (
	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc
	github.com/gin-contrib/sse v0.1.0 // indirect
	github.com/gin-gonic/gin v1.4.0 // indirect
	github.com/go-openapi/spec v0.19.2 // indirect
	github.com/go-openapi/swag v0.19.3 // indirect
	github.com/kr/pty v1.1.8 // indirect
	github.com/mailru/easyjson v0.0.0-20190626092158-b2ccc519800e // indirect
	github.com/mattn/go-isatty v0.0.8 // indirect
	github.com/swaggo/gin-swagger v1.1.0 // indirect
	github.com/swaggo/swag v1.6.0
	github.com/ugorji/go v1.1.5-pre // indirect
	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb // indirect
	golang.org/x/tools v0.0.0-20190701194522-38ae2c8f6412 // indirect
)
```

```go
// $ cat main.go
package main

import (
	"github.com/gin-gonic/gin"
	"github.com/swaggo/gin-swagger"
	"github.com/swaggo/gin-swagger/swaggerFiles"

	_ "godemo/docs" // docs is generated by Swag CLI, you have to import it.
)

// @title Swagger Example API
// @version 1.0
// @description This is a sample server Petstore server.
// @termsOfService http://swagger.io/terms/

// @contact.name API Support
// @contact.url http://www.swagger.io/support
// @contact.email support@swagger.io

// @license.name Apache 2.0
// @license.url http://www.apache.org/licenses/LICENSE-2.0.html

// @host petstore.swagger.io
// @BasePath /v2
func main() {
	r := gin.New()

	config := &ginSwagger.Config{
		URL: "http://localhost:8080/swagger/doc.json", //The url pointing to API definition
	}
	// use ginSwagger middleware to
	r.GET("/swagger/*any", ginSwagger.CustomWrapHandler(config, swaggerFiles.Handler))

	r.Run()
}
```

**Comment by easonlin404** [comment]
Please use the latest version that will resolve ur issue.
``
go get -u github.com/swaggo/gin-swagger
``

**Comment by hosszukalman** [comment]
Go get also downloads v.1.1.0 maybe because I use go modules.
```
$ go get -u github.com/swaggo/gin-swagger
go: finding github.com/PuerkitoBio/urlesc latest
go: finding golang.org/x/net latest
go: finding golang.org/x/tools latest
go: finding github.com/mailru/easyjson latest
go: finding gopkg.in/check.v1 latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/sys latest
go: finding golang.org/x/crypto latest
go: finding github.com/modern-go/concurrent latest
go: extracting github.com/swaggo/gin-swagger v1.1.0
go: extracting github.com/swaggo/swag v1.6.0
```

**Comment by hosszukalman** [comment]
It work with `go get github.com/swaggo/gin-swagger@master`

**Comment by easonlin404** [comment]
Have released [v1.2.0](https://github.com/swaggo/gin-swagger/releases/tag/v1.2.0)

**Comment by easonlin404** [comment]
Please use the latest version against master.
